sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -2237,22 +2238,40 @@ SUBROUTINE MDLFLD
 ! AQF
 !===============
 
-        if (aqfcmaq_on) then
+        if (aqf_on) then
 
            IF (IGET(994)>0) THEN
              IF (LVLS(L,IGET(994))>0) THEN
                LL=LM-L+1
 !$omp parallel do private(i,j)
                DO J=JSTA,JEND
                  DO I=ISTA,IEND
-                   GRID1(I,J) = OZCON(I,J,LL)*1000.    ! convert ppm to ppb
+                   IF(AVGOZCON(I,J,LL)<SPVAL) THEN
+                   GRID1(I,J) = AVGOZCON(I,J,LL)*1000.    ! convert ppm to ppb
```

Comment:
*1000. should be removed since AVGOZCON is already in unit of ppb

Reply:
@KaiWang-NOAA Thanks for catching this.